wordpress-mobile/gutenberg-mobile
# Correct layout within bottom-sheet [workaround]

Fixes #1404 

Ref to gutenberg PR: https://github.com/WordPress/gutenberg/pull/18574

To test:

1.Run the app on a smaller device or on the regular in horizontal mode
2.User should be able to scroll through all the blocks / add them
3.Please check also other bottom-sheets in the app e.g: Image settings

PR submission checklist:
